src/libmongoc/tests/test-mongoc-sample-commands.c

```diff
@@ -2822,6 +2847,18 @@ test_example_60 (mongoc_database_t *db)
    /* End Snapshot Query Example 2 Post */
 }
 
+// `test_snapshot_query_examples` examples for DRIVERS-2181.
+static void
+test_snapshot_query_examples (void)
+{
+   capture_logs (true);
+   test_snapshot_query_example_1 ();
+   ASSERT_NO_CAPTURED_LOGS ("test_snapshot_query_example_1");
+   test_snapshot_query_example_2 ();
+   ASSERT_NO_CAPTURED_LOGS ("test_snapshot_query_example_2");
+   capture_logs (false);
```

Comment:
```suggestion
   capture_logs (true);
   test_snapshot_query_example_1 ();
   capture_logs (false);
   ASSERT_NO_CAPTURED_LOGS ("test_snapshot_query_example_1");

   capture_logs (true);
   test_snapshot_query_example_2 ();
   capture_logs (false);
   ASSERT_NO_CAPTURED_LOGS ("test_snapshot_query_example_2");
```
Recommend consistent pattern of minimizing capture "scope" as much as possible.

Also applies to other instances of `capture_logs (false)` added in this PR.